### ISSUE TITLE
Blue bar behind status bar

### ISSUE THREAD

**Issue by mquestionable** [KBLNY/react-native-message-bar#29]
Related to #6 (which is apparently closed) and probably #19 and there is PR #27 but for different kind of issue (or maybe it's same cause but different behavior).

I have this issue too. I have set status bar background to be transparent. After message is shown and I click on TextInput so keyboard is visible, message goes up and there is blue background under status bar (until message goes away - message never goes down again even if you hide keyboard).

Are we still waiting for this fix or is this the new issue?
Thanks

**Comment by trinadhkoya** [comment]
Yeah ,i understand i.e it is automatically taking the below one,i just replaced that annoying blue color with #fff(white).You can change the color how ever you want
```
                   stylesheetInfo={{ backgroundColor: '#fff', strokeColor: '#fff' }}
``` #19
